internal/config/instancetypes/aws.go

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+//go:generate go run ../../../hack/generate-aws-instancetypes/main.go > aws.go
+// Code generated by hack/generate-aws-instancetypes tool. DO NOT EDIT.
+
+package instancetypes
+
+var AWSSupportedInstanceTypesOrFamilies = []string{"M5", "M5a", "M5ad", "M5d", "M5dn", "M5n", "M5zn", "M6a", "M6i", "M6id", "T3", "T3a", "C5", "C5a", "C5ad", "C5d", "C5n", "C6a", "C6i", "C6id", "Hpc6a", "R5", "R5a", "R5ad", "R5b", "R5d", "R5dn", "R5n", "R6a", "R6i", "R6id", "X2idn", "X2iedn", "X2iezn", "z1d", "D3", "D3en", "I3en", "I4i", "DL1", "G4", "G4ad", "G5", "Inf1", "P4", "VT1", "u-3tb1.56xlarge", "u-6tb1.56xlarge", "u-6tb1.112xlarge", "u-9tb1.112xlarge", "u-12tb1.112xlarge", "p3dn.24xlarge", "m5.metal", "m5d.metal", "m5dn.metal", "m5n.metal", "m5zn.metal", "m6a.metal", "m6i.metal", "m6id.metal", "c5.metal", "c5d.metal", "c5n.metal", "c6a.metal", "c6i.metal", "c6id.metal", "r5.metal", "r5b.metal", "r5d.metal", "r5dn.metal", "r5n.metal", "r6a.metal", "r6i.metal", "r6id.metal", "u-6tb1.metal", "u-9tb1.metal", "u-12tb1.metal", "u-18tb1.metal", "u-24tb1.metal", "x2idn.metal", "x2iedn.metal", "x2iezn.metal", "z1d.metal", "i3.metal", "i3en.metal", "i4i.metal", "g4dn.metal"}
```

Comment:
Could you add newlines to the generated output? :grinning:

Reply:
Do you mean for the user-visible printed output in the CLI or this just for this file / slice specifically?